app/utils/date_utils.py

```diff
@@ -160,7 +161,7 @@ def get_expiry_dates(
     symbol: ``str``
         The symbol that represents stock or index to which expiry dates are required.
     symbol_type: ``SymbolType``
-        The type of symbol it is, means either `equity` or `derivative`
+        The type of symbol it is, means either ``equity`` or ``derivative``
```

Comment:
only add \`` to parameter types not in their explanations
correct: symbol_type: \``SymbolType\``
The type of symbol it is, means either \``equity\`` or \``derivative\``
remove \`` from these kinds of sentences.